server/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/models/LinearModel.java

```diff
@@ -86,7 +86,16 @@ public MovAvgModel clone() {
 
     @Override
     public double next(Collection<Double> values) {
```

Comment:
Can we add a comment here explaining why this is duplicating functionality provided by `MovingFunctions`? I think someone could try to deduplicate the code without this

Reply:
Oof, actually, this should be pulled back out.  This was added when I thought it was a breaking change but it's not actually (since we don't allow null/NaN as a value because of `gap_policy`) in regular mov avg.

Thanks for noticing, I'll revert this to pointing at the static MovingFunctions  version.